tests/test_save_load.py

```diff
@@ -56,7 +56,7 @@ def test_file_method():
     """
     x = SimpleClass(6)
     with tempfile.NamedTemporaryFile() as named_file:
-        with h5py.File(named_file.name) as h5_file:
+        with h5py.File(named_file.name, 'r+') as h5_file:
```

Comment:
why this? did the default change?

Reply:
Yeah, default will be 'r' in the future.

Reply:
At least, if the deprecation warning / message is to be believed..